panels/shopping-list/ha-panel-shopping-list.html

```diff
@@ -66,70 +66,68 @@
+          <ha-start-voice-button hass='[[hass]]' can-listen='{{canListen}}'></ha-start-voice-button>
+          <paper-menu-button
+            horizontal-align="right"
+            horizontal-offset="-5"
+            vertical-offset="-5"
+          >
+            <paper-icon-button
+              icon="mdi:dots-vertical"
+              slot="dropdown-trigger"
+            ></paper-icon-button>
+            <paper-listbox slot="dropdown-content">
+              <paper-item
+                on-tap="_clearCompleted"
+              >Clear completed</paper-item>
+            </paper-listbox>
+          </paper-menu-button>
         </app-toolbar>
       </app-header>
 
       <div class='content'>
         <paper-card>
-          <div class="card-content list-header">Shopping List
-            <div class="list-options">
-              <paper-menu-button
-                horizontal-align="right"
-                horizontal-offset="-5"
-                vertical-offset="-5"
-              >
-                <paper-icon-button
-                  icon="mdi:dots-vertical"
-                  slot="dropdown-trigger"
-                ></paper-icon-button>
-                <paper-listbox slot="dropdown-content">
-                  <paper-item
-                    on-tap="_clearCompleted"
-                  >Clear completed</paper-item>
-                </paper-listbox>
-              </paper-menu-button>
-            </div>
-          </div>
-          <template is='dom-if' if='[[!items.length]]'>
-            <div class='card-content'>
-              You have no items on your shopping list.
-            </div>
-          </template>
+          <paper-icon-item on-focus='_focusRowInput'>
+            <paper-icon-button
+              slot="item-icon"
+              icon="mdi:plus"
+              on-tap='_addItem'
+            ></paper-icon-button>
+            <paper-item-body>
+              <paper-input
+                id='addBox'
+                placeholder="Add item"
+                on-keydown='_addKeyPress'
+                no-label-float
+              ></paper-input>
+            </paper-item-body>
+          </paper-icon-item>
+
           <template is='dom-repeat' items='[[items]]'>
             <paper-icon-item on-tap='_enableEditting'>
-              <paper-checkbox
-                slot="item-icon"
-                checked='{{item.complete}}'
-                on-tap='_itemCompleteTapped'
-                tabindex='0'
-              ></paper-checkbox>
+                <paper-checkbox
+                  slot="item-icon"
+                  checked='{{item.complete}}'
+                  on-tap='_itemCompleteTapped'
+                  tabindex='0'
+                ></paper-checkbox>
               <paper-item-body>
-                <template is='dom-if' if='[[!_computeIsEditting(_editIndex, index)]]'>
+                <!-- <template is='dom-if' if='[[!_computeIsEditting(_editIndex, index)]]'>
```

Comment:
Is this section needed?

Reply:
that's what I get for doing late night PRs 👍 removed.